packages/libs/react-ui/src/index.ts

```diff
@@ -122,3 +122,6 @@ export {
   maskValue,
   useDialog,
 } from './components';
+
+// re-exports from 'react-aria'
+export { RouterProvider } from 'react-aria';
```

Comment:
Can you add some documentation or examples of how this component would work with next/link?

Reply:
it is already documented here https://react-spectrum.adobe.com/react-aria/routing.html, I will add the link in storybook